tasks/libs/common/omnibus.py

```diff
@@ -312,6 +316,7 @@ def send_build_metrics(ctx, overall_duration):
                     ],
                     'unit': 'seconds',
                     'type': 0,
+                    "metadata": get_origin_metrics(17, 0, 0, True),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏it looks like we always use `True` to include the origin field. Do you know when we should not include it? If yes, could you add it to the docs in the `get_origin_metrics` function?

Reply:
In `size.py` the metrics are sent through the create_gauge() function where `origin_field=False`:
<img width="716" alt="image" src="https://github.com/user-attachments/assets/f88fca06-37a3-433a-a314-24866e94a54d">

create_gauge() ultimately calls the create_metric() function:
<img width="711" alt="image" src="https://github.com/user-attachments/assets/b4703796-42f1-4ecd-a1d5-77f859cdf612">
these metrics are generated with the datadog-api-client classes and it handles the origin structure which is why the origin_field isn't needed

I will add this to the docs of the `get_origin_metrics`

Reply:
Thanks!